examples/resources/spectrocloud_cluster_azure/providers.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_providers {
+    spectrocloud = {
+      version = ">= 0.1"
+      source  = "spectrocloud/spectrocloud"
+    }
+  }
+}
+provider "spectrocloud" {
+  host         = var.sc_host
+  username     = var.sc_username
+  password     = var.sc_password
```

Comment:
we decided to remove passwords from examples, please replace with key.

Reply:
done